### ISSUE TITLE
remote configuration vs the panel

### ISSUE THREAD

**Issue by RHGF44** [UnamSanctam/SilentCryptoMiner#1631]
Hello. Is it possible to set control in the miner through remote configuration and through the panel? Where will he take the settings in the first place?

**Comment by UnamSanctam** [comment]
Yes you can use both, the web panel will take priority over the "Remote Configuration".

**Comment by RHGF44** [comment]
1. How often does the miner access the control panel and remote configuration?
2. What happens if a panel is specified in the miner settings, but it does not work?

**Comment by UnamSanctam** [comment]
1. The miner contacts the web panel ever 1 minute, and it contacts the "Remote Configuration" ever 100 minutes (to prevent bans).
2. Then it just skips it.

**Comment by RHGF44** [comment]
1. If there is a remote configuration, then at startup, when does the miner access the remote configuration? (immediately or after 100 minutes).
2. When accessing a remote configuration, do the settings change immediately? Are these remote configuration settings stored somewhere in the miner or does the miner check the remote configuration every time it starts after 100 minutes?

**Comment by UnamSanctam** [comment]
1. Immediately.
2. Yes they change immediately, the settings are not stored permanently on the computer (there are a lot of possible issues with doing that) but are instead only retrieved on start and every 100 minutes after that.

**Comment by tarekxxx** [comment]
Bro is xmr-eu1.nanopool.org His Port is same 14433 or its changed for 10343 ?? ssl

**Comment by UnamSanctam** [comment]
>Bro is xmr-eu1.nanopool.org His Port is same 14433 or its changed for 10343 ?? ssl

They have changed the port to `10343` but `14433` still seems to work for now.
